src/stores/user.store.ts

```diff
@@ -13,12 +13,21 @@ export const useUserStore = defineStore('user', () => {
 	const ToastStore = useToastStore()
 	const ErrorStore = useErrorStore()
 	const inscriptions = ref({})
+	const MailVerified = ref (false)
 	const { setContent }  = ToastStore;
 	const { add_error } = ErrorStore;
 	async function get_csrf() {
 		const response = await axios.get('/user/get-csrf/')
 	}
 
+
+	//C'ets une dingz mais promis ça va marcher :
+	async function verifMail(idname: String , idtoken: String){
+		MailVerified.value=false
+		let result = await axios.get(`/user/confirm/${idname}/${idtoken}`)
+		MailVerified.value=true
```

Comment:
This is not very clear, add at least a comment to explain that you use default axios interceptor to leave your function in case of error.